mteb/tasks/Clustering/deu/BlurbsClusteringP2P.py

```diff
@@ -5,6 +5,8 @@
 
 
 class BlurbsClusteringP2P(AbsTaskClustering):
+    superseeded_by = "BlurbsClusteringP2P.v2"
```

Comment:
Maybe name the task "BlurbClusteringP2PFast" ?

Reply:
I like the v2 better, but it is not a strong preference. Ideally I want the AbsTaskClustering to be renamed such that the fast approach becomes the default (this is way I would prefer not calling it fast)